tidb-cloud/connect-to-tidb-cluster.md

```diff
@@ -13,23 +13,28 @@ This document introduces the methods to connect to your TiDB Dedicated cluster.
 
 After your TiDB Dedicated cluster is created on TiDB Cloud, you can connect to it via one of the following methods:
 
-- [Connect via standard connection](/tidb-cloud/connect-via-standard-connection.md)
+- Direct connections
 
-    The standard connection exposes a public endpoint with traffic filters, so you can connect to your TiDB cluster via a SQL client from your laptop. You can connect to your TiDB clusters using TLS, which ensures the security of data transmission from your applications to TiDB clusters.
+    Direct connections use the MySQL native connection system over TCP. You can connect to your TiDB Dedicated cluster using any tool that supports MySQL connections, such as the [MySQL Command-Line Client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html). TiDB Cloud also provides [SQL Shell](/tidb-cloud/connect-via-sql-shell.md), which enables you to try TiDB SQL, test out TiDB's compatibility with MySQL quickly, and administer user privileges.
```

Comment:
The meaning is correct, but it seems that "manage user privileges" is more commonly used.
```suggestion
    Direct connections use the MySQL native connection system over TCP. You can connect to your TiDB Dedicated cluster using any tool that supports MySQL connections, such as the [MySQL Command-Line Client](https://dev.mysql.com/doc/refman/8.0/en/mysql.html). TiDB Cloud also provides [SQL Shell](/tidb-cloud/connect-via-sql-shell.md), which enables you to try TiDB SQL, test out TiDB's compatibility with MySQL quickly, and manage user privileges.
```